test/parallel_api/numeric/numeric.ops/scan_2.pass.cpp

```diff
@@ -227,6 +227,20 @@ DEFINE_TEST_1(test_scan_non_inplace, TestingAlgoritm)
     }
 };
 
+namespace TestUtils
+{
+// struct for checking if iterator is heterogeneous or not
+template <typename Iter, typename Void = void> // for non-heterogeneous iterators
+struct is_hetero_iterator : ::std::false_type
```

Comment:
It seems we have it (a trait) in oneDPL headers.
Why is it duplicated in the tests?

Reply:
Fixed: this code were removed from tests.